connector/pom.xml

```diff
@@ -28,257 +28,166 @@
   <artifactId>hive-bigquery-connector</artifactId>
   <version>${revision}</version>
 
-  <licenses>
```

Comment:
please restore the license

Reply:
It should automatically get inherited from the parent, no? At least I see it in the flattened POM.